storage/store.go

```diff
@@ -1156,6 +1165,12 @@ func (s *Store) processRangeDescriptorUpdateLocked(rng *Range) error {
 	delete(s.uninitRanges, rng.Desc().RaftID)
 	s.feed.registerRange(rng, false /* scan */)
 
+	// Update the max bytes of the range after snapshot applying.
+	// If no updating of zoneConfigs for this key range, then
+	// maxBytes of this range will not be set.
+	if err := s.setRangesMaxBytesFromGossip(rng); err != nil {
```

Comment:
I think this probably belongs in Range.ApplySnapshot (in range_raftstorage.go) instead of Store.processRangeDescriptorUpdateLocked.